virttest/libvirt_xml/vm_xml.py

```diff
@@ -557,18 +557,21 @@ def __init__(self, hypervisor_type='kvm', virsh_instance=base.virsh):
         self.xml = u"<domain type='%s'></domain>" % hypervisor_type
 
     @staticmethod  # static method (no self) needed b/c calls VMXML.__new__
-    def new_from_dumpxml(vm_name, options="", virsh_instance=base.virsh):
+    def new_from_dumpxml(vm_name, options="", virsh_instance=base.virsh,
```

Comment:
Yes, I am using the uri from virsh_instance which is just part of it. But using virsh.dumpxml() directly returns string, but for performing some operations with xml object of remote machine we can use new_from_dumpxml() with uri which in turn uses virsh_instance uri.